akka-remote/src/main/scala/akka/remote/transport/netty/CustomTrustManagerFactory.scala

```diff
@@ -0,0 +1,7 @@
+package akka.remote.transport.netty
+
+import javax.net.ssl.TrustManager
```

Comment:
The `TrustManager` is not netty-specific so it doesn't strictly speaking need to be in the `akka.remote.transport.netty` package - but it makes sense to have it here I guess?

Reply:
As it was used by NettySSLSupport I put it there, I can of course move it.